elephant/spike_train_surrogates.py

```diff
@@ -147,13 +143,95 @@ def dither_spikes(spiketrain, dither, n=1, decimals=None, edges=True):
         surr = [np.sort(s[np.all([s >= tstart, s < tstop], axis=0)]) * pq.s
                 for s in surr.simplified.magnitude]
 
-    # Return the surrogates as SpikeTrains
+    # Return the surrogates as list of neo.SpikeTrain
     return [neo.SpikeTrain(s,
                            t_start=spiketrain.t_start,
                            t_stop=spiketrain.t_stop).rescale(spiketrain.units)
             for s in surr]
 
 
+def dither_spikes_with_refractory_period(spiketrain, dither, n_surrogates=1,
```

Comment:
maybe it makes sense to add `refractory_period` argument to `dither_spikes()` function as you eventually did with homogeneous Poisson generation?

Reply:
Yes, I will think about how to do this or to give it another name, it's anyway lengthy.